src/settings.py

```diff
@@ -33,6 +33,9 @@ class GlobalSettings:
     pretty_print_when_loading = False
     # Used for random seeding in test environment.
     test_env_seed_offset = 10000
+    # The method to use for segmentation. By default, segment using options.
```

Comment:
@ashayathalye note this change (we should be using `--segmenter contacts`)

Reply:
(it will crash if you don't make this change, so at least it won't silently do the wrong thing)